packages/block-library/src/cover/save.js

```diff
@@ -34,6 +36,8 @@ export default function save( { attributes } ) {
 		minHeight,
 	} = attributes;
 	const overlayColorClass = getColorClassName( 'background-color', overlayColor );
+	const gradientClass = __experimentalGetGradientClass( gradient );
```

Comment:
__experimentalGetGradientClass and getColorClassName

Some naming inconsistencies here :) Also when do you think we should make the "gradients" stable?

Reply:
> __experimentalGetGradientClass and getColorClassName

Some naming inconsistencies here :)

Hopefully, it is experimental so we can correct the inconsistency.

Also when do you think we should make the "gradients" stable?

I think first we should merge the custom gradient picker, and stabilize its UI.
It seems we may end up with tabs to separate solid colors and gradients as @mtias suggested https://github.com/WordPress/gutenberg/pull/17603#issuecomment-554507138.
When we go for that design a gradient hook may not make sense and it may be better to add gradient support in the colors hook (essentially move the hook we have now into useColors) so we can return the tabbed UI there and remove the need for blocks to add it.

Reply:
agreed about the hooks, but for the theme support, I think we can probably remove the experimental flag from there unless you think the format is subject to change?